dashboard_api/core/config.py

```diff
@@ -10,7 +10,11 @@
 
 STAGE = os.environ.get("STAGE", config_object["STAGE"])
 VECTOR_TILESERVER_URL = os.environ.get("VECTOR_TILESERVER_URL", config_object["VECTOR_TILESERVER_URL"])
```

Comment:
is this preferable to:
```python
os.environ.get("VECTOR_TILESERVER_URL", config_object.get("VECTOR_TILESERVER_URL") or "")
```

Reply:
I can change it to that -- I actually prefer that style. Though I'd probably do this instead:
```
os.environ.get("VECTOR_TILESERVER_URL", config_object.get("VECTOR_TILESERVER_URL")) or ""
```

Reply:
changed